website/pages/en/index.js

```diff
@@ -164,7 +164,7 @@ const Model = props => (
   <Block id="model">
     {[
       {
-        content: '<div class="typeset">The template model defines the structure of data that are used in the natural language and logic. Accord Project templates use the <a href="https://github.com/hyperledger/composer-concerto">Composer Concerto Modeling Language</a>.</div>',
+        content: '<div class="typeset">The template model defines the structure of data that are used in the natural language and logic. Accord Project templates use the <a href="https://github.com/accordproject/concerto">Composer Concerto Modeling Language</a>.</div>',
```

Comment:
Can you remove the reference to Concerto on this line, please?

Reply:
hello !
do you mean complete removal of `<a></a>`, or, just some hyperlink fix?

Reply:
@mttrbrts  ping :)

Reply:
I believe @mttrbrts is saying to remove the second sentence "_Accord Project templates use the ..._"

Reply:
@irmerk  well, sounds reasonable

gonna do that then